service/domain/messages/room_listAliases.go

```diff
@@ -0,0 +1,78 @@
+package messages
+
+import (
+	"encoding/json"
+
+	"github.com/boreq/errors"
+	"github.com/planetary-social/scuttlego/service/domain/refs"
+	"github.com/planetary-social/scuttlego/service/domain/rooms"
+	"github.com/planetary-social/scuttlego/service/domain/transport/rpc"
+)
+
+var (
+	RoomListAliasesProcedure = rpc.MustNewProcedure(
+		rpc.MustNewProcedureName([]string{"room", "listAliases"}),
+		rpc.ProcedureTypeAsync,
+	)
+)
+
+func NewRoomListAliases(arguments RoomListAliasesArguments) (*rpc.Request, error) {
+	j, err := arguments.MarshalJSON()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to marshal arguments")
+	}
+
+	return rpc.NewRequest(
+		RoomListAliasesProcedure.Name(),
+		RoomListAliasesProcedure.Typ(),
+		j,
+	)
+}
+
+type RoomListAliasesArguments struct {
+	identity refs.Identity
+}
+
+func NewRoomListAliasesArguments(
+	identity refs.Identity,
+) (RoomListAliasesArguments, error) {
+	if identity.IsZero() {
+		return RoomListAliasesArguments{}, errors.New("zero value of identity")
+	}
+
+	return RoomListAliasesArguments{
+		identity: identity,
+	}, nil
+}
+
+func (i RoomListAliasesArguments) MarshalJSON() ([]byte, error) {
+	return json.Marshal([]string{
+		i.identity.String(),
+	})
+}
+
+type RoomListAliasesResponse struct {
+	aliases []rooms.Alias
+}
+
+func NewRoomsListAliasesResponse(b []byte) (RoomListAliasesResponse, error) {
```

Comment:
I think we need some tests covering this function as probably it doesn't work in the current state. `json.Unmarshal` won't work with `rooms.Alias` if I'm not mistaken.